### DISCUSSION TITLE
Text mask for TextElement

### DISCUSSION THREAD

**Discussion by adamberecz** [vueform/vueform#59] [Ideas]
Text mask for TextElement

**Comment by fahrettinenes** [comment]
Extremely need it.

**Comment by glennlaysonjr** [comment]
Any update on this?

**Reply by adamberecz** [reply]
This will be out next, I'm already working on it.

**Comment by adamberecz** [comment]
🥁🥁🥁 ...text masks are out with `1.9.0` 🎉

All info and usage here:
https://vueform.com/docs/input-mask

**Reply by afuno** [reply]
Thank you so much!
